lib/src/analysis_server.dart

```diff
@@ -286,7 +300,7 @@ abstract class AnalysisServerWrapper {
           ..line = error.location.startLine
           ..message = utils.normalizeFilePaths(error.message)
           ..sourceName = path.basename(error.location.file)
-          ..hasFixes = error.hasFix
+          ..hasFixes = error.hasFix!
```

Comment:
If this property is nullable, are there any cases where this may be null that may occur in this case?

Perhaps a default value of `false` wouldn't hurt?

```suggestion
          ..hasFixes = error.hasFix ?? false
```